### PR TITLE
MegaRaid controller in Linux (Broadcom/LSI)

### PR DESCRIPTION
**Pull request by mpikzink**
Add the LSI Raid Controller mib to Linux even as preparation for a separate module. There is additional software necessary on the monitored host to extend SNMP.
 https://docs.broadcom.com/docs/1211132411799

Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/discovery/linux.yaml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+                        - { descr: disabled , graph: 0, value: 0, generic: 1 }
+                        - { descr: enabled  , graph: 0, value: 1, generic: 0 }
+                -
+                    oid: enclosurePowerSupplyTable
+                    value: powerSupplyStatus
+                    num_oid: '.1.3.6.1.4.1.3582.4.1.5.5.1.3.{{ $index }}'
+                    group: 'Enclosure {{ $enclosureId-EPST }}'
+                    descr: 'Powersupply {{ $powerSupplyID }}'
+                    state_name: powerSupplyStatus
+                    states:
```
Comment (mpikzink) [inline includes/definitions/discovery/linux.yaml:79]: Does someone knows, how to use the same states multiple times?

Comment (PipoCanaja) [reply includes/definitions/discovery/linux.yaml:79]: Hi @mpikzink 
To my knowledge, it is not (yet) possible in the YAML syntax. But all patches are welcomed 👍

Comment (mpikzink) [reply includes/definitions/discovery/linux.yaml:79]: Thanks @PipoCanaja I put it on the to-do list :-)

**Comment by murrant** [comment]
Is this a draft @mpikzink ?

**Comment by mpikzink** [comment]
Yes @murrant , I like to convert to a module because the software should in principal also work with windows systems. But my local tries with module don't work yet.

**Comment by murrant** [comment]
Test data needs to be regenerated due to upstream changes (removing ifHighSpeed)

**Review by murrant — APPROVED** [review]

**Pull request merged**